docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -34,6 +34,38 @@ image::images/cases-ui-connector.png[Shows the page for creating connectors]
 - {kibana-ref}/swimlane-action-type.html[{swimlane} connector]
 - {kibana-ref}/cases-webhook-action-type.html[{webhook-cm} connector]
 
+[float]
+[[mapped-case-fields]]
+=== Mapped case fields
+
+To represent an {es-sec} case in an external system, {es-sec} case fields are mapped to existing fields in {sn}, {jira}, {ibm-r}, and {swimlane}. Once fields are mapped, you can push updates to external systems and mapped fields are overwritten or appended. Retrieving data from external systems is not supported.
+
+|===
+
+| *Case field* | *Mapped field*
+
+| Title
+
+a| The case `Title` field is mapped to corresponding fields in external systems. Mapped field values are overwritten when you push updates.
+
+* *{sn}*: Mapped to the `Short description` field.
+* *{jira}*: Mapped to the `Summary` field.
+* *{ibm-r}*: Mapped to the `Name` field.
+* *{swimlane}*: Mapped to the `Description` field.
+
+| Description
+| The case `Description` field is mapped to the `Description` field in {sn}, {jira}, {ibm-r}, and {swimlane}. Mapped field values are overwritten when you push updates.
```

Comment:
```suggestion
| The case `Description` field is mapped to the `Description` field in all external systems. Mapped field values are overwritten when you push updates.
```

Reply:
@stephmilovic is this the case for the Webhook Case management connector too?

Reply:
Oops, didn't realize there were other external connectors

Reply:
There's just one more, and it's on me. I just remembered we added the Webhook - Case Management connector in 8.5. 

The Webhook - Case Management connector is kind of a special case because it gives users a way to connect to _any_ external case management system. Obviously, there's no way to predict or see the name of external field that'll hold the case's description (i.e., the external field that the Security case's `Description` field will map to). So, now I'm wondering if we even need to include mapping details about the  Webhook - Case Management connector here.

Reply:
@nastasha-solomon From what I know about the Webhook - Case Mgmt connector, it sounds like a special case that might not fit into the hard-coded mappings in the list/table. Maybe a quick sentence or two in the intro could establish that with the Webhook connector, the mapped fields could vary, depending on the external system, etc.?